### ISSUE TITLE
API spec endpoint contains invalid servers entry, causing OGC API validation to fail

### ISSUE THREAD

**Issue by philvarner** [jjrom/resto#258]
The API spec endpoint https://tamn.snapplanet.io/api contains a field:

```
"servers": [
    {
      "url": "http://127.0.0.1:5252",
      "description": "resto localhost server"
    }
  ],
```

When the OGC API Features Part 1 validator (https://cite.opengeospatial.org/teamengine/about/ogcapi-features-1.0/1.0/site/) is run against this, it uses the entries in `server` to run the tests (e.g., it uses the root endpoint to find the api spec, gets the api spec, and then runs everything else against the servers in the api spec instead of the root endpoint).

Since localhost isn't a valid endpoint to run this (most of the time), the test "validate Conformance Operation And Response" fails, which then cascades to skipping most of the other tests.

**Comment by jjrom** [comment]
Thanks for the feedback. This is corrected in #259

**Comment by philvarner** [comment]
I think  

```
"http://www.opengis.net/spec/ogcapi-features-1/1.0/req/oas30",
    "http://www.opengis.net/spec/ogcapi-features-1/1.0/req/geojson"
```
should also be conf instead of req. 

Also, I think resto also implements the STAC - Features conformance class, but it's not advertised via `https://api.stacspec.org/v1.0.0-beta.2/ogcapi-features`

**Comment by jjrom** [comment]
@philvarner Oooops, yes you're perfectly right :) Corrected in #265
